2.4/contrib/httpd_util.sh

```diff
@@ -0,0 +1,14 @@
+# source
+
+export HTTPD_SERVER_ADMIN=${HTTPD_SERVER_ADMIN:-default@localhost}
+
+function configure_httpd() {
+  echo "---> Configuring Apache"
```

Comment:
add " ..." here as you want till this finish. Would be also useful to explain users what is actually being configured.